.coveragerc

```diff
@@ -1,7 +1,7 @@
 # .coveragerc to control coverage.py
 [run]
 branch = True
-source = */pyscaffold/*
```

Comment:
Hi @abravalheri, not really. I what I did basically is that in case of `COVERAGE=1`, the unit tests are run again with pytest only to show the coverage correctly. I read somewhere on Stackoverflow that it is possible to map different paths to one and to completely circumvent this problem by some nifty `.coveragerc` settings. It has something to do with [paths section](https://coverage.readthedocs.io/en/coverage-4.5.1a/config.html?highlight=path#paths). Wanna look into it?